tests/AlyxPanel_test.m

```diff
@@ -461,6 +464,54 @@ function test_giveFutureWater(testCase)
         'Failed to add training dates to saved params')
     end
     
+    function test_activeFlag(testCase)
+      % Ensure that the panel is active when the DatabaseURL is added and
+      % none empty
```

Comment:
what does "and none empty" mean here?

Reply:
AlyxPanel will be inactive if A) `isfield(dat.paths, 'databaseURL') == 0` or B) `isempty(p.databaseURL)`, so if in dat.paths `p.databaseURL = '';`, then AlyxPanel will be inactive.

Reply:
why/when would `p.databaseURL = '';` ever occur though? In the `paths_template` file you already have the example as 'https://alyx.cortexlab.net'.

i.e. is it necessary to specify "none empty" ?

Reply:
It's safe to assume that at some point a user somewhere will set this field to be empty.  We don't want MATLAB to try to make requests to ''.  It's better to deal with this scenario.  Additionally there is currently no way to 'delete' a path for a specific rig, however you can override one with an empty value.  So imagine one of the rigs in the lab is being used without Alyx and is using a shared paths file, they can set the custom paths like this by saving `paths.databaseURL = [];` to paths.mat, then on only that computer the AlyxPanel will be deactivated.

Reply:
i see, ok